src/lib/HypeDayReact.tsx

```diff
@@ -78,17 +85,20 @@ export default function HypeDayReact({
         projectData={projectData}
         appId={appId}
         walletAddress={walletAddress}
+        doesTokenExist={!!userToken}
```

Comment:
I usually shorten these to tokenExist, but this is fine as well.

Reply:
I saw that I use `doesTokenExist` to identify if we have any users or not. So I changed it to `hasUser`